packages/core/src/services/utils.ts

```diff
@@ -1,7 +1,9 @@
 import { ready } from "@ribajs/utils/src/dom";
 
 /**
- * Just an Class with some helpful functions
+ * Just a class with some helpful functions
+ *
+ * TODO: Should we add all utils from other files to this class? Or just export every function by itself?
```

Comment:
Ich würde eher vorschlagen, dass man die anderen Utils da lässt wo sie sind, da sie dadurch ja irgendwie geordnet sind, oder meintest du etwas anderes?

Reply:
Okay. Ich hatte noch keinen richtigen Überblick über die Struktur des ganzen Repos.

Es gibt `riba/packages/core/src/services/util.ts`, wo diese Klasse drin ist, mit nur einer Funktion. (deprecated?)

Aber das scheint dann wohl nur für das Core-Modul intern da zu sein. Und gemeinnützige "Utility-Funktionen" gehören wohl in das Paket `@ribajs/utils` und können immer von dort importiert werden. Hatte ich nicht richtig verstanden.